util/src/main/java/io/grpc/util/AdvancedTlsX509KeyManager.java

```diff
@@ -39,20 +37,21 @@
 
 /**
  * AdvancedTlsX509KeyManager is an {@code X509ExtendedKeyManager} that allows users to configure
- * advanced TLS features, such as private key and certificate chain reloading, etc.
+ * advanced TLS features, such as private key and certificate chain reloading.
+ * We expect only one of {@link AdvancedTlsX509KeyManager#updateIdentityCredentials},
+ * {@link AdvancedTlsX509KeyManager#updateIdentityCredentialsFromFile
+ * (File, File, long, TimeUnit, ScheduledExecutorService)},
+ * {@link AdvancedTlsX509KeyManager#updateIdentityCredentialsFromFile
+ * (File, File, long, TimeUnit, ScheduledExecutorService)} methods to be called after instantiation
+ * of this class.
  */
-@ExperimentalApi("https://github.com/grpc/grpc-java/issues/8024")
 public final class AdvancedTlsX509KeyManager extends X509ExtendedKeyManager {
   private static final Logger log = Logger.getLogger(AdvancedTlsX509KeyManager.class.getName());
-
-  // The credential information sent to peers to prove our identity.
+  // Minimum allowed period for refreshing files with credential information.
+  private static final int MINIMUM_REFRESH_PERIOD = 1 ;
```

Comment:
Include "MINUTES" in the name?

Reply:
Changed to MINIMUM_REFRESH_PERIOD_IN_MINUTES